src/cheri/insns/jalr_32bit.adoc

```diff
@@ -16,56 +16,30 @@ include::base_isa_extension.adoc[]
 
 #Begin changed since last ARC review#
 
+
 Description::
 Indirect jump to the target capability in `{cs1}` with an address offset.
 +
+NOTE: The  description below contains three hooks for extending <<JALR_CHERI>> behavior, used by extensions and/or encoding formats to give a large degree of extensibility.
+ Unless an extension and/or encoding format, such as <<section_rvy_sentry_insn_ext>>,  is implemented which explicitly reference the hooks, then take no action for any of them.
++
 Operationally, copy `{cs1}` to the target <<pcc>> and then...
 +
 . Compute the target <<pcc>> address in two steps, each using the semantics of the <<SCADDR>> instruction:
 .. Increment the address of the target <<pcc>> by the sign-extended 12-bit `offset`, and
 .. set bit zero of the target <<pcc>> address to zero.
-. Clear the {ctag} of the target <<pcc>> if the triple of its <<sec_cap_type>> value, `{cd}`, and `{cs1}` is not acceptable, as defined by the capability encoding.
-+
-NOTE: All such triples are acceptable to <<rv64y_cap_description>> and <<rv32y_cap_description>>.
-. If the target <<pcc>> is a <<sentry_cap>> (for <<rv32y_cap_description>>/<<rv64y_cap_description>>: if it has <<sec_cap_type>> `1`), then unseal it.
+. HOOK 1: Optionally clear the {ctag} of the target <<pcc>>.
+. HOOK 2: Optionally unseal the target <<pcc>> if sealed.
 . If the target <<pcc>> is (still) sealed, clear its {ctag}.
-+
-NOTE: This is not possible for <<rv32y_cap_description>>/<<rv64y_cap_description>>, given its two <<sec_cap_type>> values.
 . If `{cd}≠{creg}0`, compute the return capability and install it to `{cd}`:
 .. add the width of this instruction to the current <<pcc>>, and
```

Comment:
While here, this should be amended to say "using the semantics of the `<<SCADDR>>` instruction" since the addition might, in principle, take us out of representable bounds and we need to make sure that we don't construct an incorrectly reinterpreted capability.